pkg/loki/loki.go

```diff
@@ -221,100 +220,70 @@ func (c *Config) Clone() flagext.Registerer {
 // Validate the config and returns an error if the validation
 // doesn't pass
 func (c *Config) Validate() error {
+	var errs []error
+
 	if err := c.SchemaConfig.Validate(); err != nil {
-		return errors.Wrap(err, "invalid schema config")
+		errs = append(errs, errors.Wrap(err, "CONFIG ERROR: invalid schema config"))
 	}
 	if err := c.StorageConfig.Validate(); err != nil {
-		return errors.Wrap(err, "invalid storage config")
+		errs = append(errs, errors.Wrap(err, "CONFIG ERROR: invalid storage config"))
 	}
 	if err := c.QueryRange.Validate(); err != nil {
-		return errors.Wrap(err, "invalid queryrange config")
+		errs = append(errs, errors.Wrap(err, "CONFIG ERROR: invalid queryrange config"))
 	}
 	if err := c.Querier.Validate(); err != nil {
-		return errors.Wrap(err, "invalid querier config")
+		errs = append(errs, errors.Wrap(err, "CONFIG ERROR: invalid querier config"))
 	}
 	if err := c.QueryScheduler.Validate(); err != nil {
-		return errors.Wrap(err, "invalid query_scheduler config")
+		errs = append(errs, errors.Wrap(err, "CONFIG ERROR: invalid query_scheduler config"))
 	}
 	if err := c.TableManager.Validate(); err != nil {
-		return errors.Wrap(err, "invalid tablemanager config")
+		errs = append(errs, errors.Wrap(err, "CONFIG ERROR: invalid tablemanager config"))
```

Comment:
nit so that the error message match the yaml section
```suggestion
		errs = append(errs, errors.Wrap(err, "CONFIG ERROR: invalid table_manager config"))
```